### DISCUSSION TITLE
Plotting voltage curves at 100,200,500, nth cycles

### DISCUSSION THREAD

**Discussion by kumaryash7** [pybamm-team/PyBaMM#3878] [Q&A]
I am studying degradation mechanisms and want to plot the voltage curves for 100, 300, 600, 800, 1500, and nth cycles. Is there any example notebook for it?
TIA.

**Comment by valentinsulzer** [comment]
https://docs.pybamm.org/en/stable/source/examples/notebooks/simulations_and_experiments/simulating-long-experiments.html
